### ISSUE TITLE
Upgrades from 10.2 onwards are not running phpunit tests

### ISSUE THREAD

**Issue by phil-davis** [owncloud/update-testing#33]
https://drone.owncloud.com/owncloud/update-testing/751/19/6
```
7.1: Pulling from owncloudci/php
Digest: sha256:8c43df89a63e8174c831c67ca5421b185bf31ea46fbb08b629014797582ec994
Status: Image is up to date for owncloudci/php:7.1
+ cd owncloud
+ php ./occ app:disable gallery
App directory "/drone/src/owncloud/apps-external" not found! Please put the ownCloud apps folder in the ownCloud folder or the folder above. You can also configure the location in the config.php file.+ chmod +x lib/composer/phpunit/phpunit/phpunit
+ cd tests
+ ../lib/composer/phpunit/phpunit/phpunit --configuration phpunit-autotest.xml --log-junit "autotest-results-$DATABASE.xml"
App directory "/drone/src/owncloud/apps-external" not found! Please put the ownCloud apps folder in the ownCloud folder or the folder above. You can also configure the location in the config.php file.
```

From core 10.2 I think that `apps-external` became a default dir in `apps_paths` but it does not seem to exist on install. 

Sort this out.

**Comment by phil-davis** [comment]
@davitol I think that I remember some issues with upgrading when `apps` + `apps-external` became the "default" list for `apps-paths` ?

Do you remember what had to be done? Did we have to tell installations anything special about adjusting `apps_paths` during upgrade?

**Comment by phil-davis** [comment]
I could sort out the scripting here to create the `apps-external` dir at an appropriate time in the process. But also we want to understand what is happening in the test process in this repo and make sure it is not something that badly impacts real sites that are upgrading.

**Comment by davitol** [comment]
> I could sort out the scripting here to create the `apps-external` dir at an appropriate time in the process

But 10.2 did not created you the apps-external folder?

**Comment by VicDeo** [comment]
For clean install case `apps_paths` has a default value - an array with
- non-writable `apps`
- writable `apps-external`

For any upgrade case it keeps `apps_paths` untouched

**Comment by VicDeo** [comment]
it would be good to have `config.php` and the list of directories in the owncloud root (including ownership and permissions)

**Comment by phil-davis** [comment]
Fixed by #34
